src/pybind/mgr/cephadm/services/services_map.py

```diff
@@ -0,0 +1,19 @@
+from typing import TYPE_CHECKING, Type
+
+if TYPE_CHECKING:
+    from cephadm.module import CephadmOrchestrator
+    from .cephadmservice import CephadmService
+
+registred_services: dict[str, Type["CephadmService"]] = {}
```

Comment:
@phlogistonjohn there's a circular dependency between the services_registry (the new name of the services_map) and the cephadmservice, so to avoid this but at the same time add typing annotations I'm using strings. Not sure if there's a better way to achieve that.

Reply:
Gotcha. Thanks for the reminder of this issue. This is fine with me.